gel/_internal/_save.py

```diff
@@ -1156,11 +1156,21 @@ def feed_db_data(self, obj_data: Iterable[Any]) -> None:
                 pass
 
 
+@dataclasses.dataclass(kw_only=True, frozen=True)
+class QueryRefetchArgs:
+    spec: list[tuple[
+        uuid.UUID,
+        list[tuple[bool, list[uuid.UUID]]],
```

Comment:
Separately would appreciate you creating a `TypeAliasType` for spec with a comment on its structure.